rollup.cssBundler.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs')
```

Comment:
I implemented the [first part of your first alternative](https://github.com/wikimedia/wikimedia-page-library/pull/77#discussion_r131291184):

> Importing (or requiring) ThemeTransform.css in index.js before all other imports.

I just also kept the custom rollup plugin because it meant we could add comments for each section of CSS (in the resulting file) saying which transform it originated from.